ai2thor/video_controller.py

```diff
@@ -27,7 +27,7 @@ def __init__(self,
                  **controller_kwargs):
         super().__init__(continuous=True, **controller_kwargs)
         self.step(
-            action='AddThirdPartyCamera', 
+            action='AddThirdPartyCamera',
```

Comment:
lots of small formatting cleanup, they are all good... but let me know if you dont like them